tools/deb/test/run_iptables_test.sh

```diff
@@ -6,11 +6,12 @@
 # Script requires a working docker on the test machine
 # It is run in the proxy dir, will create a docker image with proxy deb installed
 
```

Comment:
Thanks for the heads-up. I created https://github.com/istio/pilot/issues/1095 to track the issue.